mappings/net/minecraft/item/Item.mapping

```diff
@@ -117,15 +117,19 @@ CLASS net/minecraft/unmapped/C_vorddnax net/minecraft/item/Item
 	METHOD m_obcnhrty damage (Lnet/minecraft/unmapped/C_sbxfkpyv;)Z
 		COMMENT {@return whether this item can be damaged by the given {@link DamageSource source}}
 		ARG 1 source
-	METHOD m_pfwgoldc onStackClicked (Lnet/minecraft/unmapped/C_sddaxwyk;Lnet/minecraft/unmapped/C_nhvqfffd;Lnet/minecraft/unmapped/C_qcuteihm;Lnet/minecraft/unmapped/C_jzrpycqo;)Z
-		ARG 1 stack
-		ARG 2 slot
+	METHOD m_pfwgoldc onThisClickedOnStack (Lnet/minecraft/unmapped/C_sddaxwyk;Lnet/minecraft/unmapped/C_nhvqfffd;Lnet/minecraft/unmapped/C_qcuteihm;Lnet/minecraft/unmapped/C_jzrpycqo;)Z
+		COMMENT Called when an {@link ItemStack} of this item is placed on another {@link ItemStack} in a slot.
```

Comment:
Maybe it'd be good to link to the other methods with a `@see` tag

Reply:
Good idea